pallets/cash/src/tests/mod.rs

```diff
@@ -159,11 +160,12 @@ pub fn initialize_storage() {
 }
 
 pub fn validator_a_sign(data: &[u8]) -> Result<ChainSignature, Reason> {
-    std::env::set_var(
-        "ETH_KEY",
-        "6bc5ea78f041146e38233f5bc29c703c1cec8eaaa2214353ee8adf7fc598f23d",
-    );
-    validator_sign::<Test>(data)
+    let private_key =
+        hex::decode("6bc5ea78f041146e38233f5bc29c703c1cec8eaaa2214353ee8adf7fc598f23d").unwrap();
+    let secret_key = SecretKey::parse_slice(&private_key).unwrap();
+    let signature_raw = gateway_crypto::eth_sign(data, &secret_key, false);
+    let signature = ChainSignature::Eth(signature_raw);
+    Ok(signature)
```

Comment:
Hm this doesn't exercise `validator_sign` anymore though, but maybe that's ok if it's tested elsewhere?

Reply:
@jflatow do you think there is a need to change env var?

Reply:
I think that makes sense, its just like how do we test `validator_sign`?

Reply:
should it be tested at unit level? it realistically just the wrapper around sign_message --> eth_sign. It's like if we want to run tests in parallel then env vars should not be changed. 
Alternatively tests that change vars can be marked with some special attribute and run separately. But the whole test suite, there is only one such test now.